modules/admin_manual/pages/enterprise/external_storage/windows-network-drive_configuration.adoc

```diff
@@ -846,17 +760,9 @@ This is a known issue that prevents these two inter-operating correctly. Current
 
 Setups with several servers might have some difficulties in some scenarios:
 
-* The `wnd:listen` component _might_ be duplicated among several servers. This shouldn't cause a problem,
-depending on the limitations of the underlying database engine. The supported database engines should be
-able to handle concurrent access and de-duplication.
-* The `wnd:process-queue` _should_ also be able to run from any server, however limitations for concurrent
-executions still apply. As a result, you might need to serialized command execution of the
-`wnd:process-queue` among the servers (to avoid for the password lockout), which might not be possible or
-difficult to achieve. You might want to execute the command from just one specific server in this case.
-* `wnd:process-queue` + serializer. First, check the above section to know the interactions with the
-password lockout. Right now, the only option you have to set it up is to store the target file in a common
-location for all the server. We might need to provide a specific serializer for this scenario
-(based on Redis or DB)
+* The `wnd:listen` component _might_ be duplicated among several servers. This shouldn't cause a problem, depending on the limitations of the underlying database engine. The supported database engines should be able to handle concurrent access and de-duplication.
+* The `wnd:process-queue` _should_ also be able to run from any server, however limitations for concurrent executions still apply. As a result, you might need to serialized command execution of the `wnd:process-queue` among the servers (to avoid for the password lockout), which might not be possible or difficult to achieve. You might want to execute the command from just one specific server in this case.
```

Comment:
```suggestion
* The `wnd:process-queue` _should_ also be able to run from any server, however limitations for concurrent executions still apply. As a result, you might need to serialize command execution of the `wnd:process-queue` among the servers (to avoid password lockout), which might not be possible or difficult to achieve. You might want to execute the command from just one specific server in this case.
```